test/create.ts

```diff
@@ -251,14 +252,16 @@ test('async handlers', withServer, async (t, server, got) => {
 		handlers: [
 			async (options, next) => {
 				const result = await next(options);
+				// @ts-ignore Manual tests
 				result.modified = true;
 
 				return result;
 			}
-		]
+		] as HandlerFunction[]
```

Comment:
Why does this need force-casting?

Reply:
This can be resolved, it was my oversight.

Reply:
Not clear to me what is complaining, but users should not need to force cast this.